src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,7 @@
+@mixin animation-hover($property, $value) {
+  transition: $property $time-for-animation;
```

Comment:
It seems like the variable `$time-for-animation` is used here but it has not been defined within the mixin or passed as a parameter. Make sure to define it or pass it as a parameter to the mixin to avoid compilation errors. For example: `@mixin animation-hover($property, $value, $time-for-animation) { ... }`.